e2e/test/scenarios/joins/reproductions/27380-dashboard-drops-joined-fields-on-zoom-in.cy.spec.js

```diff
@@ -1,52 +1,116 @@
-import { restore, visitDashboard } from "e2e/support/helpers";
+import {
+  openQuestionActions,
+  popover,
+  queryBuilderMain,
+  restore,
+  visitDashboard,
+  mapColumnTo,
+  openColumnOptions,
+} from "e2e/support/helpers";
 import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
 
 const { ORDERS, ORDERS_ID, PRODUCTS } = SAMPLE_DATABASE;
 
-const questionDetails = {
-  query: {
-    "source-table": ORDERS_ID,
-    aggregation: [["count"]],
-    breakout: [
-      [
-        "field",
-        PRODUCTS.CREATED_AT,
-        { "source-field": ORDERS.PRODUCT_ID, "temporal-unit": "month" },
-      ],
-    ],
-  },
-  display: "line",
-};
-
-describe.skip("issue 27380", () => {
+describe("issue 27380", () => {
   beforeEach(() => {
     cy.intercept("POST", "/api/dataset").as("dataset");
 
     restore();
     cy.signInAsAdmin();
+  });
+
+  it("should not break click behavior for SQL based models #29517", () => {
```

Comment:
Were you aware that the repro for this already exists?
https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/models/reproductions/29517-native-remapped-model-drill-through-click-behavior.cy.spec.js

Reply:
Sorry, I wasn't aware, I thought the fix is just simply reverted without any repro. I didn't even bother searching the issue number in our code, that's my bad 🤦

Reply:
Removed and ensure those fixes passed locally, and it also passed on the CI in the previous CI run on this PR, since the only failed check is unit tests.